src/modules/errors/components/ApolloErrorTrace.tsx

```diff
@@ -25,8 +25,9 @@ const ApolloErrorTrace: React.FC<Props> = ({ errors }) => {
     >
       {errors.map((e) => (
         <CommonUnstyledList key={JSON.stringify(e)}>
-          {(e?.backtrace || []).map((line) => (
-            <li>{line}</li>
+          {(e?.backtrace || []).map((line, idx) => (
```

Comment:
fixes warning about keys (trying to reduce chatter in the console)

Reply:
thanks!